default.py

```diff
@@ -146,9 +147,6 @@ def get_params():
 elif mode == 117:
     Radio.ListFavourites(logged_in)
 
-elif mode == 118:
```

Comment:
It turns out all the local stations now have 320k bitrate options. So HD for all. :)